bin/open-ftl-pr.sh

```diff
@@ -18,5 +18,6 @@ docker run --rm \
     -e FLUENT_REPO_AUTH \
     -e FLUENT_L10N_TEAM_REPO \
     -e FLUENT_L10N_TEAM_REPO_BRANCH \
+    --env SITE_MODE=${SITE_MODE:-Mozorg} \
```

Comment:
NITs: Use short `-e` for consistency, or change them all. Also should quote the `"SITE_MODE=${SITE_MODE:-Mozorg}"` for the bash variable substitution.